WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -2352,12 +2352,15 @@ local function GetCustomCode(data, path)
   return data;
 end
 
-function WeakAuras.AddCodeOption(args, data, name, prefix, order, hiddenFunc, path, encloseInFunction, multipath, extraSetFunction, extraFunctions, reloadOptions)
+-- TODO find a paradigm which doesn't have five million flags for AddCodeOption so that calls to it don't always go off the screen
```

Comment:
```suggestion
-- TODO: find a paradigm which doesn't have five million flags for AddCodeOption so that calls to it don't always go off the screen
```